pre_commit/checkers/strict_paren_spacing_checker.rb

```diff
@@ -35,8 +33,10 @@ def examine_code
 
   private
 
-  def disabled_via_preference?
-    PreCommitHelper.disabled_via_preference?(HOOK_KEY, @force_pref_on)
+  def self.use_for_project?
```

Comment:
`private` (on line 34) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.